### ISSUE TITLE
Implementation: Blacklist specific app windows

### ISSUE THREAD

**Issue by szfkamil** [FelixKratz/JankyBorders#73]
In yabai, you're able to use Regex to add rules for windows of a given app.

`yabai -m rule --add label="Finder" app="^Finder$" title="(Co(py|nnect)|Move|Info|Pref)" manage=off`

Currently, you can only disable borders for a specific application only.

**Comment by Kryan90** [comment]
This would be great to have, one area it has been particularly irksome is in IDEs where the border is constantly being redrawn as various tool tip windows are activated:

Example in GoLand:
![image](https://github.com/FelixKratz/JankyBorders/assets/18740821/edad90aa-6de7-4722-bb95-e721d9865eff)

**Comment by nicocossiom** [comment]
Yes I think this is a needed addition, for example the Codex App just added a "pet" feature that shows an animated pet, since borders detects the overlay window it shows its border and makes that Codex feature unusable.

<img width="456" height="394" alt="Image" src="https://github.com/user-attachments/assets/ea3ee71d-2d62-4327-b985-c85799cb2c14" />
